### ISSUE TITLE
Bug report : Can't move your frame when you use buttons thats big enough(But playback works as expected)

### ISSUE THREAD

**Issue by Gurbuzguven** [sozi-projects/Sozi#491]
Make sure that you have read the [documentation of Sozi](http://sozi.baierouge.fr),
and especially the [installation instructions](http://sozi.baierouge.fr/pages/20-install.html)
and the [frequently asked questions](http://sozi.baierouge.fr/pages/faq.html)
before reporting an issue.
Remember that this is not a place to ask questions.
If you want to get help on Sozi, you can join the [Sozi community forum](http://sozi.baierouge.fr/community/).

Describe your problem or propose a new feature in the following sections.
Keep the headings and replace the paragraphs with your own text.

### Summary of your problem or feature request

Write a short description of your problem.

### Version of Sozi and other relevant software

* Which version of Sozi are you using?
Sozi 20.05.09, pre release of 20.08 also suffers from this problem
* Which drawing software did you use? (examples: Inkscape 0.48, LibreOffice Draw 4.2)
Inkscape 0.92.4
* On which platform? (examples: Windows 7 64-bit, Ubuntu Linux 14.04 32-bit, OS X El Capitan 64-bit)
Google Chrome OS Version 87.0.4265.0 64 bit Linux beta(Debian buster based)
* If your problem happens when playing a presentation, which browsers have you tried? (examples: Firefox 46, Chrome 52)
On playback there is no problem.

### Steps to reproduce the problem

What did you do to make the problem happen?
Making big invisible buttons in inkscape then trying move frame

Give enough details to enable us to reproduce the problem.
https://we.tl/t-gLHtcTT6dX here is the file i used(On sozi cropped partion of the frame is where the invisible button is(it's for keeping your frame from advancing that way you can add some interactivity like interactive stories using sozi) )

If you are proposing a new feature, what steps should the user follow to use it?
See above

### Expected behavior

What results did you expect?
Move your frame as expected

### Observed behavior

What results did you actually get?
Can't move your frame

### Hints and solutions (optional)

* If you have taken the time to investigate the problem, tell us what you have found.
You can move your by first zooming out using mouse wheel moving your frame when zoomed out, then adjusting to it when you zoom in you can ctrl+e it(automatically adjusts your frame when you are close enough).

* If you have ideas on how to fix the problem, please describe them.
Or you can send all your big enough buttons to a layer when adjusting your frames hide that layer etc.

**Comment by aumouvantsillage** [comment]
Let me rephrase the issue and tell me if I understand correctly.

Your document contains hyperlinks attached to big rectangles.
Pressing the left mouse button over one of these rectangles, and dragging the mouse, should move the canvas, but it does not.

I thinks it is reasonable to expect that the drag-and-drop action behaves the same over hyperlinks as anywhere else. I will investigate the problem and hopefully it will be fixed in the next version.

**Comment by Gurbuzguven** [comment]
Sorry for my confusing phrasing.Yes it's exactly like what you said.To be clear on the file i've sent before on frame one although it seems like there is only 2 hyperlinks in the frame 1, there is actually a third one which is just a big transparent rectangle covers the whole frame in the background and its hyperlinked to current frame(for example if your current frame id is "main" by hyperlinking this rectangle to "#main" will keep frame from advancing when playing, thereby only way to advance is clicking other hyperlinks)).

**Comment by aumouvantsillage** [comment]
OK. As I wrote earlier, I think this is a valid issue that needs to be fixed.

But I also have the impression that what you trying to do is already supported in Sozi without the need for invisible rectangles.

There are options in Sozi that allow to enable or disable mouse and keyboard actions selectively.
In English, one of these options is labeled "Allow to control the presentation: using the mouse / using the keyboard"
(Sunumu kontrol etmeye izin ver: Fareyi kullanarak / Klavyeyi kullanarak).
When disabled, this option will prevent moving to the next frame on mouse clicks or key presses.

**Comment by Gurbuzguven** [comment]
Thank you for your feedback but what i want to achieve is not disabling mouse alltogether, what i want is advance as expected in some frames and doesn't advance in some unless a hyperlink is clicked(Like hidden frames in powerpoint or literally frames in the long gone flash).The best example i can show you right now is in chinese unfortunately but you can still check it out on my example at http://guojunan.atwebpages.com/ (advances normally like 4-5 frames after that there is a selection screen comes you shouldn't advance unless a hyperlink is clicked)

**Comment by aumouvantsillage** [comment]
> what i want to achieve is not disabling mouse alltogether, what i want is advance as expected in some frames and doesn't advance in some unless a hyperlink is clicked

Thanks for sharing this use case and your example.
Maybe I should make an option to disable mouse actions on a per-frame basis.
However, there are other issues that need fixing first and I cannot tell when it will be done.

**Comment by Gurbuzguven** [comment]
You're an awesome developer! I also think you might focus on other aspects since presentations like this can already be done without any performance issues(Since it needs one more rectangle per frame with buttons*).But how about a guide on sozi website?If you agree i will prepare a guide for it when i have free time.

*for example my chinese lesson example has 58 frames in total but only 9 of them uses this feature.
